tests/Annotations/InMemoryStorageAdapter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace tests\eLife\Annotations;
+
+use Csa\GuzzleHttp\Middleware\Cache\Adapter\StorageAdapterInterface;
+use Csa\GuzzleHttp\Middleware\Cache\CacheMiddleware;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+final class InMemoryStorageAdapter implements StorageAdapterInterface
```

Comment:
a pity that this is not provided by https://github.com/csarrazi/guzzle-cache-middleware/tree/master/src/Adapter
I see no other choice than bundling it here for now :+1: